docs/src/main/asciidoc/firestore.adoc

```diff
@@ -0,0 +1,60 @@
+== Cloud Firestore
+
+Spring Cloud GCP provides convenience starter which automatically configures authentication settings and client objects needed to begin using the https://cloud.google.com/firestore/[Google Cloud Firestore].
+
+See https://cloud.google.com/firestore/docs/[documentation] to learn more about Cloud Firestore.
+
+=== Dependency Setup
+
+To begin using this library, add the `spring-cloud-gcp-starter-data-firestore` artifact to your project.
+
+Maven coordinates, using <<getting-started.adoc#_bill_of_materials, Spring Cloud GCP BOM>>:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-starter-data-firestore</artifactId>
```

Comment:
Why "data-firestore"?
I would just go with "spring-cloud-gcp-starter-firestore".

Reply:
It's for symmetry with spring-cloud-gcp-starter-data-datastore and spring-cloud-gcp-starter-data-spanner